packages/code-studio/src/dashboard/linker/LinkerUtils.tsx

```diff
@@ -98,13 +139,14 @@ class LinkerUtils {
    * @param {string} cloneId Cloned panel id
    * @returns {object[]} Cloned links array or empty array if no new links added
    */
-  static cloneLinksForPanel(links, panelId, cloneId) {
-    const clonedLinks = [];
+  static cloneLinksForPanel(
+    links: Link[],
+    panelId: string,
+    cloneId: string
+  ): Link[] {
+    const clonedLinks = [] as Link[];
```

Comment:
Could use `const clonedLinks: Link[] = [];`. Again not aware of any difference for either syntax

Reply:
Yea I don't care too much one way or the other. I guess having the type explicitly after the variable name is clearer than inferring the type with an `as` later